pygfx/renderers/wgpu/meshshader.py

```diff
@@ -308,9 +396,18 @@ def code_vertex(self):
 
             varyings.pick_id = u32(pick_id);
             varyings.pick_idx = u32(face_index);
-            var arr_pick_coords = array<vec3<f32>, 3>(vec3<f32>(1.0, 0.0, 0.0), vec3<f32>(0.0, 1.0, 0.0), vec3<f32>(0.0, 0.0, 1.0));
-            varyings.pick_coords = vec3<f32>(arr_pick_coords[sub_index]);
-
+            if {{indexer}} == 3 {
+                var arr_pick_coords = array<vec3<f32>, 3>(vec3<f32>(1.0, 0.0, 0.0), vec3<f32>(0.0, 1.0, 0.0), vec3<f32>(0.0, 0.0, 1.0));
+                varyings.pick_coords = vec3<f32>(arr_pick_coords[sub_index]);
+            } else {
+                var arr_pick_coords = array<vec3<f32>, 6>(vec3<f32>(1.0, 0.0, 0.0),
+                                                          vec3<f32>(0.0, 1.0, 0.0),
+                                                          vec3<f32>(0.0, 0.0, 1.0),
+                                                          vec3<f32>(1.0, 0.0, 0.0),
+                                                          vec3<f32>(0.0, 1.0, 0.0),
+                                                          vec3<f32>(0.0, 0.0, 1.0));
+                varyings.pick_coords = vec3<f32>(arr_pick_coords[sub_index_ori]);
```

Comment:
I think we'd need changes elsewhere to make picking work for quads.

Reply:
I updated _wgpu_get_pick_info for quads